spec/unit/pdk/util/puppet_version_spec.rb

```diff
@@ -167,38 +202,45 @@
       include_context 'is not a package install'
       include_context 'with a mocked rubygems response'
 
+      def result(version)
+        {
+          gem_version:  Gem::Version.new(version),
+          ruby_version: PDK::Util::RubyVersion.default_ruby_version,
```

Comment:
Is this a test of the `default_ruby_version`, too, or are you using it here only as a shortcut?

Reply:
`PDK::Util::RubyVersion.default_ruby_version` will return the details of the running ruby version for non-packaged installs (gem or git clone), so I guess shortcut?